### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by jhkd-kevin** [xuancong84/mteval#1]
you code is running on python2?

**Comment by xuancong84** [comment]
Yes, it was very old. I am no longer working in machine translation. Will find sometime to change to Python3, should be easy.
